pkgs/top-level/all-packages.nix

```diff
@@ -37738,6 +37738,8 @@ with pkgs;
 
   EmptyEpsilon = callPackage ../games/empty-epsilon { };
 
+  en-croissant = callPackage ../games/en-croissant { };
```

Comment:
new packages should be added into pkgs/by-name 

you can read more about it https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md